tests/db/compose.yml

```diff
@@ -47,7 +47,7 @@ services:
     image: mcr.microsoft.com/mssql/server
     restart: always
     environment:
-      ACCEPT_EULA: Y
+      ACCEPT_EULA: "Y"
```

Comment:
https://github.com/microsoft/mssql-docker/issues/199

Reply:
Why do we need double quotes? `ACCEPT_EULA: Y` is equivalent to `ACCEPT_EULA: "Y"`, isn't it?